Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -754,7 +754,7 @@ std::optional<LayoutUnit> RenderFlexibleBox::computeMainAxisExtentForFlexItem(Re
     }
 
     auto mainAxisWidth = isColumnFlow() ? availableLogicalHeight(AvailableLogicalHeightType::ExcludeMarginBorderPadding) : contentBoxLogicalWidth();
-    return flexItem.computeLogicalWidthUsing(sizeType, size, mainAxisWidth, *this) - flexItem.borderAndPaddingLogicalWidth();
+    return std::max<LayoutUnit>(flexItem.computeLogicalWidthUsing(sizeType, size, mainAxisWidth, *this) - flexItem.borderAndPaddingLogicalWidth(), 0);
```

Comment:
I think the point of the assert it to catch bugs like this when `computeMainAxisExtentForFlexItem` returns an unexpected logical width for the flex item....which in this case is a `<table>`, so maybe `RenderTable's` `computeLogicalWidth` is doing something odd here.
It looks like the assumption here is that `computeLogicalWidthUsing` returns the border box width and since we are interested in the content box, we immediately subtract `flexItem.borderAndPaddingLogicalWidth`. I wonder what `RenderTable` returns when the `<table>` is totally empty whether border/padding is included or not.